cypress/support/pages/checkout.pageObject.js

```diff
@@ -0,0 +1,71 @@
+import PageObject from '../PageObject';
+import faker from 'faker';
+
+const order = {
+  name: faker.name.firstName(),
```

Comment:
Order object is global and can introduce state-related issues. Consider moving it inside the class or function where it's being used.